src/DynamoCoreWpf/DynamoCoreWpf.csproj

```diff
@@ -1258,13 +1256,10 @@
     <Content Include="d3d10warp.dll">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
-    <Content Include="d3dcompiler_44.dll">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-    <Content Include="sharpdx_direct3d11_1_effects_x64.dll">
+    <Content Include="sharpdx_direct3d11_effects_x64.dll">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </Content>
-    <Content Include="sharpdx_direct3d11_effects_x64.dll">
+    <Content Include="sharpdx_direct3d11_effects_x86.dll">
```

Comment:
I think this is just some merge weirdness related to packages. Both versions of that dll are definitely still in the project and both are set to `Copy Always`